src/api/status/public/css/terminal.css

```diff
@@ -14,6 +14,7 @@
 canvas {
   width: 100% !important;
   max-height: 100%;
+  padding-left: 1rem;
```

Comment:
Why not apply the paddings for all sides

Reply:
I think padding on the left or all sides doesn't matter in this situation because all the contents already start from the left. What would you think? The `padding-left` is fine, or all sides would be better?

Reply:
@tcvan0707 it's up to your design choice. I think the text should not be so close to the edges.

Reply:
@tcvan0707 I just rethink it and adding extra padding for all side seem to be better